rs/nns/governance/proto/ic_nns_governance/pb/v1/governance.proto

```diff
@@ -2622,3 +2626,31 @@ message FinalizeDisburseMaturity {
   // The account identifer to which to transfer the ICPs.
   ic_ledger.pb.v1.AccountIdentifier to_account_identifier = 5;
 }
+
+// An ICRC-3-like value.
+message Value {
+  oneof value {
+    bytes blob = 1;
+    string text = 2;
+    bytes nat = 3;
+    bytes int = 4;
```

Comment:
These fields contain Candid-encoding of Nat and Int Candid types, right? I feel like this (embarrassing) fact needs to be called out in comments. Otherwise, how does one know how to use these fields correctly?